src/kiota/KiotaHost.cs

```diff
@@ -416,6 +416,10 @@ private static Command GetGenerateCommand()
         classOption.AddAlias("-c");
         classOption.ArgumentHelpName = "name";
         AddStringRegexValidator(classOption, classNameRegex(), "class name");
+        
+        var classAccessModifierOption = new Option<string>("--class-access-modifier", "The class access modifier to use for the client classes.");
+        classAccessModifierOption.AddAlias("-cam");
```

Comment:
```suggestion
        classAccessModifierOption.AddAlias("--cam");
```

Reply:
should we name this type access modifier so it can be applied to enums and interfaces as well?